pkg/runtime/wfengine/activity.go

```diff
@@ -100,6 +101,10 @@ func (a *activityActor) InvokeMethod(ctx context.Context, actorID string, method
 
 	// The actual execution is triggered by a reminder
 	err := a.createReliableReminder(ctx, actorID, nil)
+	if err == nil {
+		// Activity reminder created, record metrics
+		diag.DefaultWorkflowMonitoring.RemindersTotalCreated(ctx, "Activity")
```

Comment:
Don't use the string "Activity" directly. A constant can be defined in `pkg/diagnostics/workflow_metrics.go` with this value and that can be used here.

Reply:
fixed